todo_app/app.py

```diff
@@ -1,36 +1,67 @@
 from flask import Flask, render_template, url_for, redirect, request
-from todo_app.data.session_items import get_items, add_item, save_item, get_item, remove_item
+from todo_app.data.session_items import get_item
+from todo_app.data.trello_items import get_items, add_item, change_item_status, remove_item, Item, get_list_names
+from dotenv import load_dotenv
+import os
 
 
 from todo_app.flask_config import Config
 
+load_dotenv()
+
+board_id = os.getenv("BOARD_ID")
+apikey = os.getenv("APIKEY")
+token = os.getenv("TOKEN")
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
+list_targets = get_list_names(board_id, apikey, token)
+
 
 @app.route('/')
 def index():
-    items = get_items()
-    return render_template("index.html", items=items)
+    items = get_items(board_id, apikey, token)
+    items = [Item.from_trello_cards(item) for item in items]
+    #clean date in each item if they have due date
+    for item in items:
+        if item.due_date:
+            item.clean_date()
+
+    return render_template("index.html", items=items, list_targets=list_targets)
 
 
 @app.route('/create_new_task', methods=["POST"])
 def create_new_task():
-    new_item = request.form.get('todo')
-    add_item(new_item)
+    item_title = request.form.get('todo')
+    item_description = request.form.get("description")
+    item_due_date = request.form.get("date")
+    add_item(item_title,
+             item_description,
+             item_due_date,
+             apikey,
+             token)
     return redirect(url_for("index"))
 
 
 @app.route('/complete_item/<id>')
 def complete_item(id):
-    item_to_complete = get_item(id)
-    item_to_complete["status"] = "Completed"
-    save_item(item_to_complete)
+    #destination list is hardcoded id
```

Comment:
You should avoid hard coding the list ID - it means that the code is tightly coupled to a specific Trello board.

Can you get the ID from the list name `To Do` or `Doing`? This should be possible with the response from the boards API.